common/containers/Tabs/ScheduleTransaction/sagas/transaction/current/currentScheduleTimezone.ts

```diff
@@ -0,0 +1,25 @@
+import { setScheduleTimezone } from 'actions/transaction/actionCreators/fields';
+import { call, put, takeLatest } from 'redux-saga/effects';
+import { SagaIterator } from 'redux-saga';
+import { TypeKeys } from 'actions/transaction/constants';
+import { SetScheduleTimezoneAction } from 'actions/transaction';
+import { SetCurrentScheduleTimezoneAction } from '../../../actions/transaction/actionTypes/scheduleTimestamp';
+
+export function* setCurrentScheduleTimezone({
+  payload: raw
+}: SetCurrentScheduleTimezoneAction): SagaIterator {
+  let value: string | null = null;
+
+  value = raw;
```

Comment:
Please declare it straight away as value = raw, instead of first value = null and then value = raw.